src/libData/AccountData/AccountStoreSC.tpp

```diff
@@ -446,7 +448,9 @@ bool AccountStoreSC<MAP>::UpdateAccounts(const uint64_t& blockNum,
       return false;
     }
 
-    this->IncreaseBalance(fromAddr, gasRefund);
+    if (!this->IncreaseBalance(fromAddr, gasRefund)) {
+      LOG_GENERAL(WARNING, "IncreaseBalance failed for gasRefund");
```

Comment:
What is the criterion for WARNING or FATAL?

Reply:
Comparing to the others which are impossible to happen, this one is possible. If during a series of  chain call in one transaction, another contract transfers its balance to this sender, which may cause the balance being larger than the moment when the gasDeposit was not deducted, such may cause overflow issue.